ampersand-dom-bindings.js

```diff
@@ -5,13 +5,18 @@ var matchesSelector = require('matches-selector');
 var partial = require('lodash.partial');
 var slice = Array.prototype.slice;
 
-function getMatches(el, selector) {
+function getMatches(el, selector, firstOnly) {
     if (selector === '') return [el];
     var matches = [];
-    if (matchesSelector(el, selector)) matches.push(el);
-    return matches.concat(slice.call(el.querySelectorAll(selector)));
+    if(!selector) return matches;
+    if(firstOnly) {
```

Comment:
Done, thank you!

I rechecked the switch case and added another test to verify.